py/picca/fitter2/utils.py

```diff
@@ -0,0 +1,117 @@
+
+import scipy as sp
+from numpy import fft
+from scipy import special
+import scipy.interpolate
+
+from . import myGamma
+
+nmuk = 1000
+muk=(sp.arange(nmuk)+0.5)/nmuk
+dmuk = 1./nmuk
+muk=muk[:,None]
+
+def sinc(x):
+    return sp.sin(x)/x
```

Comment:
why not using scipy.sp?
It is more stable in x~=0

Reply:
A while ago I did profiling tests (with the cProfile module)  on the fitter. The results:
  * the fitter spends half the time calculating the sinc function, so even more than the fft (which is almost the rest of the time)
  * I've done tests a while ago at Utah and found that sinc was slower than sin(x)/x by about 50%, probably because it tests whether the argument is zero. 
 * We never evaluate sinc at exactly zero

I plan to redo some profiling tests at nersc, but for now I prefer to leave my sinc function.

Reply:
ok, thnk you for the answer